README.md

```diff
@@ -1,9 +1,5 @@
 # ark_automate
 
-[![GitHub stars](https://img.shields.io/github/stars/bptlab/ark_automate)](https://github.com/bptlab/ark_automate)
```

Comment:
Why has this been removed? Not that I think this is necessary, but I am just curious as to why it was removed.

Reply:
We don't know.  I just put them back in.